angular-workspace/projects/ni/nimble-angular/README.md

```diff
@@ -33,14 +33,16 @@ The steps to use components from Nimble Angular are similar to using components
     class AppModule {}
     ```
 
-3. Each application should add the `<nimble-theme-provider>` element to `app.component.html` and set its `theme` attribute. The theme provider has no appearance of its own but defines tokens that are used by descendant components.
+3. Each application should add the `<nimble-theme-provider>`* element to `app.component.html` and set its `theme` attribute. The theme provider has no appearance of its own but defines tokens that are used by descendant components.
 
     ```html
     <nimble-theme-provider theme="light">
         <router-outlet></router-outlet>
     </nimble-theme-provider>
     ```
 
+    *See [note below about the recommended way to reference nimble elements in HTML](#specifying-nimble-element-tags).
```

Comment:
I think this note and the note below are over-emphasizing the discussion. Angular Templates should always use the name and directive for typing. There is no edge case related to Angular templates.

The use-case being documented is specifically a non Angular Template workflow which is very much an exception.

I'd prefer to avoid specifying this note in the getting started section at all to avoid confusion.